src/metabase/driver.clj

```diff
@@ -386,7 +386,9 @@
     ;; DEFAULTS TO TRUE.
     :basic-aggregations
 
-    ;; Does this driver support standard deviation and variance aggregations?
+    ;; Does this driver support standard deviation and variance aggregations? Note that if variance is not supported
+    ;; directly, you can calculate it manually by taking the square of the standard deviation. See the MongoDB driver
+    ;; for example.
```

Comment:
I did take stats in college, but I completely forgot how to calculate variance given standard deviation. So I had to do a bit of Googling to figure out how to do it. This is just a note to future people who need to do it the hard way too